services/user.py

```diff
@@ -0,0 +1,40 @@
+from db.models import User
+
+
+def create_user(username: str,
+                password: str,
+                email: str = None,
+                first_name: str = None,
+                last_name: str = None) -> None:
+    user = User.objects.create_user(username=username, password=password)
+    if email:
+        user.email = email
+    if first_name:
+        user.first_name = first_name
+    if last_name:
+        user.last_name = last_name
+    user.save()
+
+
+def get_user(user_id: int) -> User:
+    return User.objects.get(id=user_id)
+
+
+def update_user(user_id: int,
+                username: str = None,
+                password: str = None,
+                email: str = None,
+                first_name: str = None,
+                last_name: str = None) -> None:
+    user = User.objects.get(id=user_id)
+    if username:
+        user.username = username
+    if password:
+        user.set_password(password)
+    if email:
+        user.email = email
```

Comment:
```suggestion
        user.email = email if email is not None else user.email
```
Теж рекомендація просто

Reply:
due to zen of Python
"Explicit is better than implicit."

Reply:
Можеш без тернарного оператора, але з None треба звірити, бо можуть передати пустий рядок, наприклад